From 8eabed00b395d9363ff5b519dd064bfcb5ef1957 Mon Sep 17 00:00:00 2001
From: Dorian Krause <d.krause@fz-juelich.de>
Date: Mon, 3 Apr 2017 10:57:01 +0200
Subject: [PATCH] test2.13: Increase sleep time

---
 testsuite/expect/test2.13 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/expect/test2.13 b/testsuite/expect/test2.13
index db3d075..4753994 100755
--- a/testsuite/expect/test2.13
+++ b/testsuite/expect/test2.13
@@ -104,7 +104,7 @@ if {[wait_for_job $job_id "RUNNING"] != 0} {
         send_user "\nFAILURE: waiting for job to start running\n"
         exit 1
 }
-sleep 5
+sleep 20
 
 #
 # Validate that step's original time limit
-- 
2.9.3

